### ISSUE TITLE
Add support for emulex ocs_fc_lio 

### ISSUE THREAD

**Issue by rlenkala** [agrover/rtslib-fb#72]
Hi,

So far we used with standard "rtslib" by providing spec file in /var/target/fabric/  to configure our Emulex OCS LIO targets. Now we wanted to use ocs_fc_lio kernel module driver with "rtslib-fc" module. We added code to support Emulex ocs_fc_lio driver in "fabric.py"  and verified the code changes. Could you please review the attached patch and let me know if any modifications are required?

I also wanted to know the code check-in

 process, I'm not sure if i can do commit at this time or not

Thanks,
Ravindar
[rtslib-fb.txt](https://github.com/agrover/rtslib-fb/files/197376/rtslib-fb.txt)

**Comment by agrover** [comment]
Hi Ravindar,

Very good to see this! We would need to wait to consider adding this until OCS LIO target is supported in the kernel.

**Comment by agrover** [comment]
Yes. This will ensure we can test it properly now, and also in the future with future rtslib-fb changes.

**Comment by 824380210** [comment]
does anybody know that emulex ocs_fc_lio  can be use in rhels 7.3 or centos 1611 ? can anybody give me some example to use it , I want to use it in target mode to test the HBA card

**Comment by rlenkala** [comment]
Hi,

ocs_fc_lio driver will supports from 3.18 kernel on wards and doesn't
matter which OS it is & ocs_fc_lio driver is not yet open-sourced.

Thanks,
Ravindar
